code/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -456,10 +540,11 @@
     "task: trunk_workers = 1, trunk_size=job_size, walltime = walltime, mem = mem, cores = numThreads, tags = f'{step_name}_{_output:bn}'\n",
     "bash: container = container, expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
     "        ${'bcftools norm -m-any' if not bi_allelic else 'bcftools view -m2 -M2'} ${'-v snps' if snp_only else \"\"} ${_input} |\\\n",
-    "        bcftools norm --check-ref w -f ${reference_genome}  -Oz|\\\n",
+    "        bcftools norm -d exact -N --check-ref w -f ${reference_genome}  -Oz --threads ${numThreads} |\\\n",
```

Comment:
I just noticed there are two `bcftools norm` commands here. Will the first `norm` perform the indel normalization we dont like? Or it's fine since `-f` is not provided? I'm not sure why we need two `norm` anyways. @hsun3163 please document what each command does if you review and decide this is the right thing to do.